lib/omnibus/software.rb

```diff
@@ -786,7 +786,7 @@ def with_standard_compiler_flags(env = {}, opts = {})
           }
         else
           {
-            "LDFLAGS" => "-Wl,-rpath,#{install_dir}/embedded/lib -L#{install_dir}/embedded/lib",
+            "LDFLAGS" => "-Wl,-rpath,#{install_dir}/embedded/lib,-z,origin -L#{install_dir}/embedded/lib",
```

Comment:
Might be adding a comment about what `-z origin` does (at the very least worth a mention on the PR description).